package.json

```diff
@@ -80,7 +82,9 @@
     "prettier": "^3.1.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-focus-on": "^3.9.2",
```

Comment:
This is a one-stop-shop for applying good a11y to modal UI

https://github.com/theKashey/react-focus-on